### ISSUE TITLE
Fail to Install sfpowerscripts in a 'M1' powered Mackbook

### ISSUE THREAD

**Issue by anthonygiuliano** [dxatscale/sfpowerscripts#864]
**Describe the bug**
Error installing @dxatscale/sfpowerscripts

```
Installing plugin @dxatscale/sfpowerscripts... failed
    Error: yarn add @dxatscale/sfpowerscripts@latest --non-interactive
    --mutex=file:/Users/anthonygiuliano/.local/share/sfdx/yarn.lock
    --preferred-cache-folder=/Users/anthonygiuliano/Library/Caches/sfdx/yarn --check-files exited with code
    1
```

**To Reproduce**
run `sfdx plugins:install @dxatscale/sfpowerscripts`

**Expected behavior**
sfpowerscripts plugin should install successfully

**Screenshots**

**Platform Details (please complete the following information):**
 - OS: macOS
 - Version: 12.2.1
 - Salesforce CLI(sfdx cli) Version: sfdx-cli/7.139.0
 - CI Platform: n/a

**Additional context**
Full output is below, along with sfdx plugins / version info.  Let me know if there's any other information that would be helpful.

```
sfdx plugins:install @dxatscale/sfpowerscripts
This plugin is not digitally signed and its authenticity cannot be verified. Continue installation y/n?: y
Finished digital signature check.
warning ../../../package.json: No license field
warning @dxatscale/sfpowerscripts > puppeteer@7.1.0: Version no longer supported. Upgrade to @latest
warning @dxatscale/sfpowerscripts > @dxatscale/sfpowerscripts.core > @salesforce/ts-sinon > sinon > @sinonjs/formatio > samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam
warning "sfpowerkit > ts-node@9.1.1" has unmet peer dependency "typescript@>=2.7".
warning Workspaces can only be enabled in private projects.
error /Users/anthonygiuliano/.local/share/sfdx/node_modules/puppeteer: Command failed.
Exit code: 1
Command: node install.js
Arguments:
Directory: /Users/anthonygiuliano/.local/share/sfdx/node_modules/puppeteer
Output:
The chromium binary is not available for arm64.
If you are on Ubuntu, you can install with:

 sudo apt install chromium


 sudo apt install chromium-browser

/Users/anthonygiuliano/.local/share/sfdx/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserFetcher.js:115
                    throw new Error();
                    ^

Error
    at /Users/anthonygiuliano/.local/share/sfdx/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserFetcher.js:115:27
    at FSReqCallback.oncomplete (node:fs:198:21)
Installing plugin @dxatscale/sfpowerscripts... failed
    Error: yarn add @dxatscale/sfpowerscripts@latest --non-interactive
    --mutex=file:/Users/anthonygiuliano/.local/share/sfdx/yarn.lock
    --preferred-cache-folder=/Users/anthonygiuliano/Library/Caches/sfdx/yarn --check-files exited with code
    1
```

```
❯ sfdx --version --verbose
 CLI Version :
	sfdx-cli/7.139.0

 Architecture:
	darwin-arm64

 Node Version :
	node-v16.13.2

 Plugin Version:
	@oclif/plugin-autocomplete 0.3.0 (core)
	@oclif/plugin-commands 1.3.0 (core)
	@oclif/plugin-help 3.3.1 (core)
	@oclif/plugin-not-found 1.2.6 (core)
	@oclif/plugin-plugins 1.10.11 (core)
	@oclif/plugin-update 1.5.0 (core)
	@oclif/plugin-warn-if-update-available 1.7.3 (core)
	@oclif/plugin-which 1.0.4 (core)
	@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
	alias 1.2.1 (core)
	apex 0.9.0 (core)
	auth 1.8.1 (core)
	community 1.1.4 (core)
	config 1.3.19 (core)
	custom-metadata 1.0.12 (core)
	data 0.6.9 (core)
	generator 1.2.2 (core)
	info 1.2.1 (core)
	limits 1.3.0 (core)
	org 1.11.1 (core)
	salesforce-alm 53.10.2 (core)
	schema 1.1.0 (core)
	sfdmu 4.12.9
	sfdx-cli 7.139.0 (core)
	sfpowerkit 4.1.5
	source 1.8.12 (core)
	telemetry 1.4.0 (core)
	templates 53.6.0 (core)
	trust 1.1.0 (core)
	user 1.7.1 (core)

 OS and Version:
	Darwin 21.3.0
```

**Comment by azlam-abdulsalam** [comment]
@anthonygiuliano  Are you on an Apple  having M1 by any chance?

**Comment by anthonygiuliano** [comment]
I am.  I figured it had something to do with that.  Is it a known issue already?

**Comment by azlam-abdulsalam** [comment]
I had some folks ping me about it, looks like the version of puppeteer we are running is not compatible.   We will investigate this further

**Comment by azlam-abdulsalam** [comment]
@anthonygiuliano There is couple of workarounds listed here https://github.com/puppeteer/puppeteer/issues/6622

 Looks  like its fixed in https://github.com/puppeteer/puppeteer/pull/7099, Will be updating the dependencies

**Comment by azlam-abdulsalam** [comment]
Fixed in 10.4.2

**Comment by anthonygiuliano** [comment]
Thanks so much, I'm able to install on my M1 after the April release!
